test/cmd/newapp.sh

```diff
@@ -14,6 +14,48 @@ trap os::test::junit::reconcile_output EXIT
 os::util::environment::setup_time_vars
 
 os::test::junit::declare_suite_start "cmd/newapp"
+
+#
+# imagestream/tag creation and reuse
+#
+os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json -n openshift'
+os::cmd::expect_success 'oc delete imagestreamtag php:latest -n openshift'
+os::cmd::expect_success 'oc new-project test-imagestreams'
+os::cmd::try_until_failure 'oc get imagestreamtag php:latest -n openshift'
+
+# should fail due to missing php:latest tag
+os::cmd::expect_failure 'oc new-app --image-stream=openshift/php https://github.com/sclorg/cakephp-ex'
+
+# should succeed and create the php:latest tag in the current namespace
+os::cmd::expect_success 'oc new-app --docker-image=library/php https://github.com/sclorg/cakephp-ex --strategy=source'
+os::cmd::try_until_success 'oc get imagestreamtag php:latest -n test-imagestreams'
+os::cmd::expect_success 'oc create imagestreamtag php:latest -n openshift'
```

Comment:
this is valid?  what does it do? (what does php:latest point to?)

Reply:
fixed in next push